### PR TITLE
Add test for "Self Terms of Service Review"

### PR DESCRIPTION
**Pull request by kurtismullins**

### CODE REVIEW

**Comment by chentex** [comment]
Access issues... this will outplay us.

File: pkg/tests/tests.go [hunk 1]
```diff
@@ -84,6 +84,14 @@ var tests = []helpers.TestOptions{
 		Method:   http.MethodPost,
 		Handler:  handlers.TestClusterAuthorizations,
 	},
+	{
+		TestName:    "self-terms-review",
+		Path:        "/api/authorizations/v1/self_terms_review",
+		Method:      http.MethodPost,
+		Body:        handlers.EmptyBody(),
```
Comment (chentex) [inline pkg/tests/tests.go:91]: I think you can just not assign it and it will do the same, most of the other request work like that.

**Review by chentex — COMMENTED** [review]
Comment aside, LGTM

File: pkg/tests/tests.go (hunk 1 above)
Comment (kurtismullins) [reply pkg/tests/tests.go:91]: Good point! I did this in haste. Let me remove this.

**Review by chentex — APPROVED** [review]
LGTM

**Pull request merged**